mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown

```diff
@@ -98,8 +98,8 @@ immediately, with more details in dedicated headers below:
 `terraform plan` or `terraform apply` on Google provider `5.0.0` or later with
 an existing pre-`5.0.0` resource before an `apply`, the plan will show an
 update adding your current `labels` values to `terraform_labels`. This change
-may result in a no-op update call to the API depending on the resource
-implementation, but can otherwise be safely applied.
+may result in a no-op update call to the API or resource recreation depending
```

Comment:
We should work out our plan for `StateUpgraders` before this change. Additionally, I wouldn't say that the "can be safely applied" bit applies to resource recreation!

Reply:
I updated the description of this PR with the resources that need state upgrades. Is it expected?